internal/services/cdn/cdn_endpoint_custom_domain_resource.go

```diff
@@ -87,6 +87,7 @@ func resourceArmCdnEndpointCustomDomain() *pluginsdk.Resource {
 							Type:     pluginsdk.TypeString,
 							Optional: true,
 							ValidateFunc: validation.StringInSlice([]string{
+								string(cdn.MinimumTLSVersionNone),
```

Comment:
the type `None` should be implied by the property being omitted, so we don't need to add this constant value and can instead default this in the Create/Update and Read?

Reply:
That is also possible, however, the schema is currently using the `TLS12` as the default value, changing the default value is a breaking change. Or we can do the change as we are approacing 3.0?

Reply:
@magodo fair point, these resources are sufficiently different enough we should probably split these by CDN Type (since as a number of issues highlight, when using a Verizon CDN you must use a separate portal to manage it) instead?

Reply:
TBH, currently there is no official doc about which sku supports which profile (heard from the service team).

Reply:
Yeah it'd need manual work, but we should be able to determine those from the bug reports/trial and error: https://github.com/hashicorp/terraform-provider-azurerm/labels/service%2Fcdn

At this point though I think that's probably the best path forward here, as these are completely different CDNs with different rules applied to them?

Reply:
The only difference fwik is the settings inside `cdn_managed_https` for CDN managed https, i.e. the  following 3 properties: `certificate_type`, `protocol_type` and `tls_version`. I don't think it worth to create 4 different resources for different skus (of cdn endpoint). Probably some of them share the same logic at the end. Also, the service team might evolve gradually to make different skus converge at some point in the future.

Reply:
I have run into the same issue but with user_managed_https using Verizon CDN.  When will the fix be released?

Reply:
@magodo the default of 1.2 makes sense for the other cases - these have needed splitting for a while, but I think now's probably the time to do it since this change is forcing a default change (and we're generally working to remove `None` across the provider in favour of omitting the field, since these are semantically the same in Terraform)

Reply:
I think my latest mmit has done the change, i.e. removing the `None` infavor of omitting the field. Please check and let me know what others needed.